src/System.Runtime.Extensions/src/System/Collections/Comparer.cs

```diff
@@ -17,7 +17,6 @@
 
 namespace System.Collections
 {
-    [Serializable]
```

Comment:
There didn't seem to be a problem found by the tests, but I agree that's fishy. I'll enable this and its dependencies, CompareInfo and SortVersion.